rfcs/XXX-eresource_authentication/README.md

```diff
@@ -0,0 +1,59 @@
+# E-Resource Authentication
+
+Wellcome Collection like many other institutions offers its staff and patrons access to electronic journals & books offered by online databases/publishers.
+
+The list of databases offered is [available on the Wellcome Collection site](https://wellcomecollection.org/pages/YDaP2BMAACUAT7DS).
+
+The "Collections and Research" team handles relationships with publishers / database providers, negotiates terms of access and communicate changes in access requirements.
+
+As part of decommissioning the Wellcome Library site we must ensure continuity of access to those resources for patrons and staff.
+
+## E-resources in the Catalogue API
+
+E-resources are currently available from the page linked above, but [can also be found](https://api.wellcomecollection.org/catalogue/v2/works?query=awa6c6gm&include=items) in the catalogue API.
+
+Items will have an `online-resource` location:
+
+![online resource item](online_resource.png)
+
+The URL indexed is for the [EBSCO](https://www.ebsco.com/) Link Resolver
+                           
+EBSCO is a service used by "Collections and Research" to handle database/e-journal metadata.
+
+## Current access methods
+
+### Staff
+
+Access for staff is via [Shibboleth](https://en.wikipedia.org/wiki/Shibboleth_Single_Sign-on_architecture), a profile of [SAML](https://en.wikipedia.org/wiki/Security_Assertion_Markup_Language).
```

Comment:
Gonna have to explain shibboleth somewhere with a request diagram and also discuss federations and publisher requirements for membership - athens/openathens and the relationship to shibboleth.